server/challenges/Provider.ts

```diff
@@ -1,6 +1,14 @@
 import { EventEmitter } from 'events'
+import { Challenge } from './types'
 
 export interface Provider extends EventEmitter {
   forceUpdate (): void;
+  /*
```

Comment:
(style nitpick) we should add a blank line between functions

Reply:
I feel like style concerns should be formalized in `yarn lint` or ignored.